faunadb-java/src/main/java/com/faunadb/client/response/Token.java

```diff
@@ -0,0 +1,47 @@
+package com.faunadb.client.response;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.faunadb.client.types.LazyValue;
+import com.faunadb.client.types.Ref;
+import com.google.common.base.Joiner;
+import com.google.common.collect.ImmutableMap;
+
+public final class Token extends Instance {
+  @JsonProperty("credentials")
+  private final Ref credentials;
+  @JsonProperty("secret")
+  private final String secret;
+
+  @JsonCreator
+  Token(@JsonProperty("ref") Ref ref,
+        @JsonProperty("class") Ref classRef,
+        @JsonProperty("ts") Long ts,
+        @JsonProperty("credentials") Ref credentials,
```

Comment:
ill add it once the docs are up to ensure a stable interface. nvidia doesnt need it anyways